relayer/cli/src/commands/query/client.rs

```diff
@@ -8,11 +8,20 @@ use relayer::config::{ChainConfig, Config};
 //use relayer_modules::ics02_client::query::QueryClientFullState;
 use relayer_modules::ics24_host::identifier::ClientId;
 
-//use crate::commands::utils::block_on;
+use crate::commands::utils::block_on;
 use relayer::chain::tendermint::TendermintChain;
+use relayer::query::{query, Request};
+use relayer_modules::error::Error;
 use relayer_modules::ics24_host::error::ValidationError;
+use relayer_modules::path::{ClientConnectionsPath, Path};
+use std::str::FromStr;
+use tendermint::abci::Path as TendermintPath;
 use tendermint::chain::Id as ChainId;
 
+// *****************************************************************
```

Comment:
Following our last call, was wondering if we can have more idiomatic comments. I like the stars `**` here and how well they stand out (we used them in TLA+ specs also), but not sure if this is typical for Rust codebases. Perhaps we can get inspired from:

https://doc.rust-lang.org/rust-by-example/meta/doc.html
https://github.com/informalsystems/tendermint-rs/blob/master/light-client/src/fork_detector.rs

Reply:
Sure, should have used the '///' comment because that will help with docs. I'll fix that.

Reply:
Fixed in b4dfcf3cd8466ed4652a6ca9af88e200de3e2533